### DISCUSSION TITLE
Comic/Book Name Editing

### DISCUSSION THREAD

**Discussion by arcanescroll** [Kareadita/Kavita#2902] [Ideas]
### Idea Description

Being able to edit the name of a comic would be great.  This field is greyed out and often contains the file name for some reason.  Localized Name doesn't seem to be used anywhere, so changing that makes no difference.  

### Idea Category

Web UI

### Duration of Using Kavita

1 year

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by illotum** [comment]
For reference, #2153.

Agree wholeheartedly. I have a number of manga issues with typos in, or simply badly written, ComicInfo. Despite being in the same folder, Kavita splits such files into separate entries. No clue how to merge those volumes into a single title by metadata alone. Despite [competitor] being significantly slower, I am forced to use it because it lets me merge those volumes into a single title.

Editing cbz is a non-starter. It is a skill, and while I fixed up several offenders, one cannot ask customer to spend time on it.

And finally, good design would be to do content-addressable storage with checksums for file ID and complete disregard for names. Happy to contribute if the maintainers agree.

**Comment by tyrgoossens** [comment]
I understand having a unique identifier field that is not user-editable. Just the option to display "localised name" in the interface instead would do a lot already. I would like to tag my manga correctly, but prefer the english titles over the romanised ones.

**Reply by majora2007** [reply]
Localized name does show in the UI  if it's set.

**Reply by tyrgoossens** [reply]
I should have specified: while the "localised name" is shown on the series page, when browsing a library the "name" is shown under the cover preview for a series. This is a very minor thing as I can set the "sort name" to be the same as the "localised name" but I landed on this page looking for a setting to change it.

<img width="246" alt="Screenshot 2024-05-12 at 16 35 28" src="https://github.com/Kareadita/Kavita/assets/14939903/59a1cb2f-4060-4b14-b3d3-27083c257048">
<img width="1014" alt="Screenshot 2024-05-12 at 16 35 48" src="https://github.com/Kareadita/Kavita/assets/14939903/b251fa9d-5c62-4dc4-8035-d003eb90c9d8">

**Reply by majora2007** [reply]
Ahh okay, I understand. You might want to make a separate feature request for this. There is a big UX refresh coming (still being designed) and having the community's input on stuff like this is important. More ideas might spawn as well.

**Comment by majora2007** [comment]
This is now available in the nightly (v0.9.0.21) and will be in the upcoming Stable (v0.9.1)
